static/src/javascripts/projects/common/modules/experiments/tests/newsletter-embed-test.js

```diff
@@ -23,25 +38,34 @@ export const newsletterEmbeds: ABTest = {
             id: 'variant',
             test: (): void => {
                 const iframes = ((document.querySelectorAll('.email-sub__iframe'): NodeList<any>): NodeList<HTMLIFrameElement>);
-                iframes.forEach(
-                    (ifrm) => {
-                        const ifrmElement = (ifrm: HTMLIFrameElement)
-                        const doc = ifrmElement.contentDocument ? ifrm.contentDocument : ifrm.contentWindow.document;
-                        if (doc) {
-                            const oldDesign = doc.querySelector('.js-ab-embed-old-design');
-                            const newDesign = doc.querySelector('.js-ab-embed-new-design');
-                            if (oldDesign && newDesign) {
-                                oldDesign.classList.add("hide-element")
-                                newDesign.classList.remove("hide-element")
-
+                iframes.forEach( (ifrm: HTMLIFrameElement) => {
+                    if (ifrm.id !== 'footer__email-form') {
+                        const doc = ifrm.contentDocument ? ifrm.contentDocument : ifrm.contentWindow.document;
+                        window.addEventListener('load', () => {
+                            if (doc) {
+                                const oldDesign = doc.querySelector('.js-ab-embed-old-design');
+                                const newDesign = doc.querySelector('.js-ab-embed-new-design');
+                                if (oldDesign && newDesign) {
+                                    oldDesign.classList.add("hide-element");
+                                    newDesign.classList.remove("hide-element")
+                                }
+                                trackComponentInOphan(ifrm.id, 'variant');
```

Comment:
Maybe put this inside the if statement just above it, although `oldDesign` and `newDesign` should always exist once the iframe loads - unless there's some kind of error.